main.go

```diff
@@ -11,24 +12,28 @@ import (
 	"github.com/ONSdigital/dp-api-clients-go/filter"
 	"github.com/ONSdigital/dp-download-service/config"
 	"github.com/ONSdigital/dp-download-service/service"
-	"github.com/ONSdigital/go-ns/log"
+	"github.com/ONSdigital/log.go/log"
 )
 
+var BuildTime, GitCommit, Version string
```

Comment:
These should have comments above them to indicate what they are, something like [this](https://github.com/ONSdigital/dp-frontend-filter-dataset-controller/pull/188/files#diff-7ddfb3e035b42cd70649cc33393fe32cR23)